lua/starfall/libs_sh/hologram.lua

```diff
@@ -293,14 +293,28 @@ else
 		end
 	end
 
+	local render_GetColorModulation, render_GetBlend = render.GetColorModulation, render.GetBlend
+	local render_SetColorModulation, render_SetBlend = render.SetColorModulation, render.SetBlend
+
```

Comment:
If you really need to localize those, it should be done outside of the instance environment.

Reply:
Eh, Starfall's codebase is really inconsistent with this stuff, sometimes it localizes in the instance, other times outside, so I guess it's fine.